external/llvm-project/mlir/test/Target/LLVMIR/rocdl.mlir

```diff
@@ -368,16 +368,6 @@ llvm.func @rocdl.wmma(%arg0 : vector<8xf32>, %arg1 : vector<16 x f16>, %arg2 : v
   llvm.return %r0 : vector<8xf32>
 }
 
-llvm.func @rocdl.wmma.fp8(%arg0 : vector<2 x i32>, %arg1 : vector<8xf32>) -> vector<8xf32> {
```

Comment:
Why'd this test get removed?

Reply:
Sure, it was duplicated. I guess this test in upstream PR was in a different line.